doc/hardware/peripherals/i2c.rst

```diff
@@ -6,50 +6,41 @@ I2C
 Overview
 ********
 
-.. note::
-
-   Zephyr recognizes the need to change the terms "master" and "slave"
-   used in the current `I2C Specification <i2c-specification>`_.  This
-   will be done when the conditions identified in
-   :ref:`coding_guideline_inclusive_language` have been met.  Existing
-   documentation, data structures, functions, and value symbols in code
-   are likely to change at that point.
-
```

Comment:
I think it would beneficial to our readers to keep some sort of clarification to the terminology used within the Zephyr I2C controller/target APIs. People unfamiliar with the recent change of terminology in the I2C specification can have difficulties understanding it otherwise.

Reply:
This makes sense, I can add a new note pointing out that the terminology has been updated and redefine what a controller and target actually mean. I have no interest in directly mapping the old terminology to the new terminology, as that continues the propagation of it.